From a472a66f8c575d569e38905dc09bfcdfae8a1c1e Mon Sep 17 00:00:00 2001
Message-ID: <a472a66f8c575d569e38905dc09bfcdfae8a1c1e.1710967273.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1710967273.git.sharlatanus@gmail.com>
References: <cover.1710967273.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Tue, 5 Mar 2024 22:10:54 +0000
Subject: [PATCH 05/49] gnu: python-pytest-doctestplus: Update to 1.2.0.

* gnu/packages/python-check.scm (python-pytest-doctestplus): Update to 1.2.0.
[arguments] <#:test-flags>: Disable one more failing test.
[home-page]: Update accordingly with lint warning.

Change-Id: I353a671741cf887c348d627d9e071e3fedcc4edc
---
 gnu/packages/python-check.scm | 11 +++++++----
 1 file changed, 7 insertions(+), 4 deletions(-)

diff --git a/gnu/packages/python-check.scm b/gnu/packages/python-check.scm
index 51a38bbcbe..99f52b6846 100644
--- a/gnu/packages/python-check.scm
+++ b/gnu/packages/python-check.scm
@@ -604,26 +604,29 @@ (define-public python-pytest-cookies
 (define-public python-pytest-doctestplus
   (package
     (name "python-pytest-doctestplus")
-    (version "1.0.0")
+    (version "1.2.0")
     (source
      (origin
        (method url-fetch)
        (uri (pypi-uri "pytest-doctestplus" version))
        (sha256
-        (base32 "17ylfnrcvvp6sd13bfj40jl40paqmjsbywysszb3xqgdr86l8l7n"))))
+        (base32 "0cmrkgpib869kpy8h8hfkg20w16lakkmbkw8cxdywpmf5wx7dbf5"))))
     (build-system pyproject-build-system)
     (arguments
      (list #:test-flags
            #~(list "-k" (string-append
+                         ;; Tests requiring network access.
                          "not test_remote_data_url"
                          " and not test_remote_data_float_cmp"
                          " and not test_remote_data_ignore_whitespace"
                          " and not test_remote_data_ellipsis"
                          " and not test_remote_data_requires"
-                         " and not test_remote_data_ignore_warnings"))))
+                         " and not test_remote_data_ignore_warnings"
+                         ;; Requiring git available.
+                         " and not test_generate_diff_basic"))))
     (native-inputs
      (list python-numpy python-pytest python-setuptools-scm))
-    (home-page "https://github.com/astropy/pytest-doctestplus")
+    (home-page "https://github.com/scientific-python/pytest-doctestplus")
     (synopsis "Pytest plugin with advanced doctest features")
     (description
      "This package contains a plugin for the Pytest framework that provides
-- 
2.41.0

